### ISSUE TITLE
gap with results in papers

### ISSUE THREAD

**Issue by BenjaminGit001** [thohemp/6DRepNet#14]
Hi,
Thanks for your impressive paper and code. I tried this repo to reproduce this performance, I followed all instruction and trained on300w-lp use train.py without change any parameters, then evaluate on AFLW2000 using test.py and results as below:

**me:** Yaw: 3.9897, Pitch: 5.0923, Roll: 3.6405, MAE: 4.2408
**yours:** Yaw: 3.63 , Pitch: 4.91      Roll:  3.37 , MAE:  3.97

Is there any other tricks or changes should be apply for reproduce your results?

**Comment by thohemp** [comment]
Hi,

the reported results are generated with the same code and setup from this repo. You can try adjusting the learning rate or scheduler (e.g. decreasing the decay from .5 to .1) to check if this improves the results.

There are multiple factors like different hardware, model initialization and random sample loading that can lead to slightly different results. Especially the last two points will induce randomness to each training.

**Comment by thohemp** [comment]
@BenjaminGit001 I updated the default training settings. This should help you to achieve the reported results more easily (or in my trainings even surpass them). Let me know if you still struggle after using them.

**Comment by zxcvbnm12316** [comment]
> 您好， 感谢您令人印象深刻的论文和代码。我尝试使用此存储库来重现此性能，我遵循所有说明并使用 train.py 在 300w-lp 上进行训练，而不更改任何参数，然后使用 test.py 对 AFLW2000 进行评估，结果如下：
> 
> **我：**偏航：3.9897，俯仰：5.0923，滚动：3.6405，MAE：4.2408 **你的：**偏航：3.63，俯仰：4.91 滚动：3.37，MAE：3.97
> 
> 是否需要应用任何其他技巧或更改来重现您的结果？

@BenjaminGit001 你好，我最近尝试使用此存储库来重现此性能，在 train.py 在 300w-lp 上进行训练，不更改任何参数，然后使用 test.py 对 AFLW2000 进行评估，结果如下
Yaw: 35.6810, Pitch: 42.4646, Roll: 24.0692, MAE: 34.0716，
但论文中的结果偏航：3.63，俯仰：4.91 滚动：3.37，MAE：3.97，这个问题已困扰我许久，请问您是否知道是什么原因吗，非常感谢您！

**Comment by zxcvbnm12316** [comment]
> Hi, Thanks for your impressive paper and code. I tried this repo to reproduce this performance, I followed all instruction and trained on300w-lp use train.py without change any parameters, then evaluate on AFLW2000 using test.py and results as below:
> 
> **me:** Yaw: 3.9897, Pitch: 5.0923, Roll: 3.6405, MAE: 4.2408 **yours:** Yaw: 3.63 , Pitch: 4.91 Roll: 3.37 , MAE: 3.97
> 
> Is there any other tricks or changes should be apply for reproduce your results?

Hello, I retrained the model and got the ‘’epoch.tar‘’ file. The pressurized file contains the ‘’data.pkl‘’ file, and the instance file shows the need for.‘’pth‘’ file. How do I get the**.pth_** file for snapshots,thank you。

Required in the document：
python test.py --batch_size 64 \ --dataset AFLW2000 \ --data_dir datasets/AFLW2000 \ --filename_list datasets/AFLW2000/files.txt \ --snapshot output/snapshots/1.pth \ --show_viz False

I trained to get the files
_epoch_30 .tar
Unzip the file, which contains the following three files
data data.pkl vwesion
